.gitignore

```diff
@@ -1,5 +1,5 @@
 .idea
-.env
+
```

Comment:
czemu usuwasz .env z git ignore?

Reply:
Zobacz, po nitce do kłębka :)
Tu na screenie poniżej masz wygenerowany „automatycznie” plik — poprzez „createReactApp". 
Co jest w nim charakterystyczne? Nie ma w nim pliku `.env`, a jest plik `.env.local`. Możemy pomyśleć, czy to błąd, jednakże nic bardziej mylnego.
W takim razie sprawdźmy czym różnią się te dwa pliki ze zmiennymi środowiskowymi:
https://stackoverflow.com/questions/55690143/what-is-the-difference-between-env-local-and-env-development-local

W dużym skrócie. Projekt zwykle potrzebuje pliku `.env` — zapiszemy w nim między innymi informację o adresach do API.
Natomiast, jeśli do danego API potrzebujemy klucza, też możemy dodać do `.env.local`. I nie będziemy tego wypychać na server.
Podobna sytuacja spotka np adres/ustawienia dla bazy danych.
Na lokalnej wersji bedziemy chcieli korzystać z lokalnego mongoDB `.env.local` , publiczna strona skorzysta już z np z chmury `.env`. A jeśli bedziemy chcieli odpalić test to zapewne skorzystamy `.env.test.local`.

Plik  `.env` jest nadpisywany przez pozostałe :)


https://github.com/PiotrRynio/PiotrRynio-PiotrRynio-CodersCamp2021.Project-3.TypeScript/blob/6761ba591af9adf31121ed6e99fea3076a642937/.gitignore

![image](https://user-images.githubusercontent.com/41823689/164895624-6815ca0b-8ae7-49b6-a304-6b6ac5ce6cc1.png)